examples/moodle-presslabs-stash/namespace1/moodle1.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   plugins: ["profilecohort"]
   mySQLServiceName: cluster1-mysql-master
   mySQLUserName: root
-  mySQLUserPassword: mypass
+  mySQLPassword: cluster1-secret.ROOT_VALUE
```

Comment:
Shouldn't this be:

mySQLPassword: cluster1-secret.ROOT_PASSWORD?

The data section in cluster1-secret has ROOT_PASSWORD as the key and not ROOT_VALUE.

https://github.com/cloud-ark/kubeplus/blob/master/examples/moodle-with-presslabs/namespace1/cluster1-secret.yaml#L9

Reply:
Ah, yes probably another reason that the code did not work. Good catch